### PR TITLE
feat(plugin): add webhookTags

### PR DESCRIPTION
**Pull request by Cynosphere**

### CODE REVIEW

File: src/plugins/webhookTags.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            },
+        },
+        {
+            find: '.BOT=0]="BOT"',
+            replacement: {
+                match: /case (.)\.BOT:default:(.)=/,
+                replace: (orig, types, text) =>
+                    `case ${types}.WEBHOOK:${text}="WEBHOOK";break;${orig}`,
+            },
+        },
```
Comment (Ruthenic) [inline src/plugins/webhookTags.ts:24]: you can merge these into one patch object (`replacement` can be an array)

**Review by Ruthenic — CHANGES_REQUESTED** [review]

File: src/plugins/webhookTags.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        {
+            find: ".Types.ORIGINAL_POSTER",
+            replacement: {
+                match: /return null==(.)\?null:.\.createElement\((.)\.Z/,
```
Comment (Vendicated) [inline src/plugins/webhookTags.ts:28]: is hardcoding Z safe?

Comment (adryd325) [reply src/plugins/webhookTags.ts:28]: .default always turns to Z
.default when there are multiple exports turns to ZP

Comment (botatooo) [reply src/plugins/webhookTags.ts:28]: you never know when that might change though

Comment (adryd325) [reply src/plugins/webhookTags.ts:28]: It's consistent across every module we've seen when moved to swc

Comment (Cynosphere) [reply src/plugins/webhookTags.ts:28]: There are modules dating before the main client switched to swc that have `Z` as `default`, namely RPC.

**Review by adryd325 — APPROVED** [review]

**Review by Vendicated — APPROVED** [review]

**Pull request merged**